external-crates/move/crates/move-compiler/src/shared/mod.rs

```diff
@@ -1083,3 +1083,56 @@ pub mod known_attributes {
         }
     }
 }
+
+//**************************************************************************************************
+// Binop Processing Macro
+//**************************************************************************************************
+
+macro_rules! process_binops {
```

Comment:
Can you doc comment a macro? specifically the args? Could be helpful

Reply:
Sure, I added one hopefully explaining everything.